documentation/API-authentication-token.md

```diff
@@ -0,0 +1,12 @@
+# API authentication token
+
+< to be described >
+
+jwt token with as minimum the following attributes:
+
+* scopes
+* party identity
+* school identity (optional)
+
+## Validity of the token
+The token will be valid for at least 30 minutes
```

Comment:
Edwin is talking about setting up changes within 30 minutes - does this not conflict with a JWT token which will be valid for more then 30 minutes (thus resulting in the old behaviour if the consents are included in the token)